dation-woocommerce.php

```diff
@@ -10,7 +10,7 @@
  * Description: Je website altijd up-to-date met je Dation planning. Importeer je Dation cursussen als Woocommerce producten.
  * Author: Dation
  * Author URI: http://www.dation.nl
- * Version: 1.1.2
+ * Version: 1.2.0
```

Comment:
Does upgrading this plugin for someone with a TKM website not break stuff?

Reply:
Yes, however, updates are still done manually(by Chris/Jordy), so with proper communication this will not be a problem.  Also, this will only happen for this specific update